honeybee_radiance/dynamic/group.py

```diff
@@ -289,6 +291,46 @@ def states_json_list(self):
                 })
         return states_list
 
+    def rfluxmtx_control_params(self, sampling=None, up_direction=None):
+        """Get Rfluxmtx control parameters.
+
+        If the values are not provided they will be assigned based on the first State
+        of the dynamic object.
+
+        Args:
+            sampling: Set hemisphere sampling type. Acceptable inputs for hemisphere
+                sampling type are:
+                * u for uniform.(Usually applicable for ground).
+                * kf for klems full.
+                * kh for klems half.
+                * kq for klems quarter.
+                * rN for Reinhart - Tregenza type skies. N stands for subdivisions and
+                defaults to 1.
+                * scN for shirley-chiu subdivisions.
```

Comment:
You need line breaks around this bulleted list for it to render correctly in Sphinx.